app/services/vot/component_value.rb

```diff
@@ -1,5 +1,31 @@
 # frozen_string_literal: true
 
 module Vot
-  ComponentValue = Data.define(:name, :description, :implied_component_values, :requirements).freeze
+  ComponentValue = Data.define(
+    :name, :description, :implied_component_values, :requirements
+  ) do
+    # @param name [String] - unique identifier
+    # @param description [String]
+    # @param implied_component_values [Array<String>]
+    # @param requirements [Array<Symbol>] see Vot::REQUIREMENTS
+    def initialize(name:, description:, implied_component_values: [], requirements: [])
+      super(
+        name: name.freeze,
+        description: description.freeze,
+        implied_component_values: implied_component_values.freeze,
+        requirements: requirements.freeze
+      )
+    end
+  end.freeze
+
+  REQUIREMENTS = {
```

Comment:
Where is this `REQUIREMENTS` constant used?

Reply:
No where, I can remove this. I just needed to map the requirements in my head.